stdnum/de/stnr.py

```diff
@@ -90,28 +92,28 @@ def _clean_region(region):
 
 class _Format():
 
-    def __init__(self, fmt):
+    def __init__(self, fmt: str) -> None:
         self._fmt = fmt
         self._re = re.compile('^%s$' % re.sub(
             r'([FBUP])\1*',
             lambda x: r'(\d{%d})' % len(x.group(0)), fmt))
 
-    def match(self, number):
+    def match(self, number: str) -> re.Match[str] | None:
         return self._re.match(number)
 
-    def replace(self, f, b, u, p):
+    def replace(self, f: str, b: str, u: str, p: str) -> str:
         items = iter([f, b, u, p])
         return re.sub(r'([FBUP])\1*', lambda x: next(items), self._fmt)
 
 
 # Convert the structure to something that we can easily use
 _number_formats_per_region = dict(
```

Comment:
Is there a nicer solution for this other than leaving both `_number_formats_per_region_raw` and `_number_formats_per_region` around?